torch/ao/quantization/pt2e/utils.py

```diff
@@ -261,13 +264,41 @@ def _is_literal(arg):
         with gm.graph.inserting_after(last_ph):
             new_args = []
             for arg in node.args:
-                if _is_literal(arg):
-                    new_args.append(gm.graph.placeholder("arg" + str(cnt)))
-                    gm._in_spec.children_specs[0].children_specs.append(LeafSpec())
-                    cnt += 1
+                if _is_literal(arg) and not arg in exclude_literals:
+                    if merge_dup and arg in literal_to_ph:
+                        new_args.append(literal_to_ph[arg])
+                    else:
+                        ph_node = gm.graph.placeholder("arg" + str(cnt))
+                        new_args.append(ph_node)
+                        gm._in_spec.children_specs[0].children_specs.append(LeafSpec())
+                        cnt += 1
+                        if merge_dup:
+                            literal_to_ph[arg] = ph_node
                 else:
                     new_args.append(arg)
             new_args = tuple(new_args)
 
         node.args = new_args
     return gm
+
+
+def replace_literals_with_existing_placeholders(gm, exclude_literals, literal_to_ph_idx):
```

Comment:
I don't follow why we need this in addition to the original `replace_literals_with_placeholders`. Can you explain in a comment how this is different from `replace_literals_with_new_placeholders` and the motivation for this? Also what do the args do?

Reply:
sure, added some docs